gnome-extra/gnome-software/gnome-software-43.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson virtualx xdg
+
+DESCRIPTION="Gnome install & update software"
+HOMEPAGE="https://wiki.gnome.org/Apps/Software https://gitlab.gnome.org/GNOME/gnome-software"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="flatpak +firmware gnome gtk-doc sysprof udev"
+RESTRICT="test" # See TODO below
+
+RDEPEND="
+	>=dev-libs/appstream-0.14.0:0=
+	>=x11-libs/gdk-pixbuf-2.32.0:2
+	>=dev-libs/libxmlb-0.1.7:=
+	>=gui-libs/gtk-4.6:4
+	>=dev-libs/glib-2.66.0:2
```

Comment:
I see this in the `pkgdiff -b` output:

````diff
-glib = dependency('glib-2.0', version : '>= 2.66.0')
+glib = dependency('glib-2.0', version : '>= 2.70.0')
````

so we should raise the required version to 2.70.0.

Reply:
In fact, a little farther down, I see this:

````diff
+# This should be available in GLib 2.74
+if meson.get_compiler('c').has_header_symbol('glib.h', 'G_FORMAT_SIZE_ONLY_VALUE', dependencies: glib)
+  conf.set('HAVE_G_FORMAT_SIZE_ONLY_VALUE', '1')
+endif
````

If the comment is accurate, we may want to just raise the dependency straight to 2.74.0.

Reply:
@mattst88 should we stick to >= 2.70.0 as 2.74.0 is not yet available